srcpkgs/aide/template

```diff
@@ -0,0 +1,26 @@
+# Template file for 'aide'
+pkgname=aide
+version=0.16.2
+revision=1
+build_style=gnu-configure
+configure_args="--with-mhash --with-posix-acl --with-prelink --with-xattr
+ --with-zlib --with-e2fsattrs --disable-static"
+conf_files="/etc/aide.conf"
+make_dirs="/var/lib/aide 0750 root root
+ /var/log/aide 0750 root root"
+hostmakedepends="flex bison"
```

Comment:
`flex` depends on `bison`, `bison` is redundant.

Reply:
Was avoiding implicit dependencies. From the README:

> o GNU flex.
> o GNU yacc (bison).

Reply:
Fixed.